tests/integration/api_v3/rpc/blockchainApps.test.js

```diff
@@ -31,8 +31,17 @@ const {
 
 const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
 const getBlockchainApps = async (params) => request(wsRpcUrl, 'get.blockchain.apps', params);
+const getNetworkStatus = async params => request(wsRpcUrl, 'get.network.status', params);
 
+let curChainID;
+
+// TODO: Update test when data is available in blockchain_apps table
```

Comment:
```suggestion
```

Reply:
The table is currently empty and we would need to tweak the tests once data is available. Should we still remove it?

Reply:
Yes, both keeping/removing them is fine. Either way, we'll get to know that the tests need to be fixed when we have the environment setup and the tests are running on the CI pipeline.

Reply:
Will prefer to keep the TODO then as we are doing same for other tests.